homeassistant/components/overkiz/cover_entities/vertical_cover.py

```diff
@@ -152,7 +152,7 @@ def __init__(
     ) -> None:
         """Initialize the device."""
         super().__init__(device_url, coordinator)
-        self._attr_name = f"{self._attr_name} Low Speed"
+        self._attr_name = "Low speed"
```

Comment:
```suggestion
        self._attr_name = "{self.name} Low speed"
```

Reply:
I thought about this as well, but with the new entity scheme, I think that it is desired to not include the device name in the `_attr_name` anymore?

Reply:
Looked at the code and `has_entity_name` has been set properly so it's as supposed to.